lib/identity_config.rb

```diff
@@ -51,11 +51,17 @@ def self.build_store(config_map)
     config.add(:aamva_verification_url)
     config.add(:account_reset_token_valid_for_days, type: :integer)
     config.add(:account_reset_wait_period_days, type: :integer)
+    config.add(:acuant_maintenance_window_start)
+    config.add(:acuant_maintenance_window_finish)
```

Comment:
we should add a type: `Time` or someting that parses these!

Reply:
I'll add a `type: :timstamp` converter

Reply:
yay! and this will need an `allow_nil` option I think if we don't have it already

Reply:
My current plan was to set it to values in the past instead of nil. But I can throw together a separate PR if we think it is time to add `allow_nil`

Reply:
Actually, it is not that complicated, let me see what I can do

Reply:
Might not be worth holding up this PR, might be easier to follow up separate --- IIRC there was also some timezone shenanigans with these particular values too

Reply:
Yeah, one interesting thing I'm running into is we haven't actually set `Time.zone` when we build the store. That said, the current timestamps have a timezone so they still work.